smos-docs-site/pages/concepts.markdown

```diff
@@ -0,0 +1,70 @@
+---
+title: SMOS Concepts
+date: 2020-04-08
+---
+
+GTD
+: GTD is a time management method by David Allen (see [Getting This
+  Done](https://en.wikipedia.org/wiki/Getting_Things_Done) on
+  Wikipedia).
+  SMOS is supposed to be a tool to implement GTD.
+
+
+<!-- the next concepts are related to SMOS configuration, but AFAIK
+are not explained -->
+
+Workflow
+: What's the meaning in SMOS and why the need for a directory?
+
+
+Projects
+: Whats the meaning in SMOS?  Why the need for a directory in the
```

Comment:
It's a subdirectory for your projects within the smos directory, so that you can also have non-project smos files like reference files, review files, archived files etc.

Reply:
As I've been playing around, I've put all my tasks in a single `test.smos` file, with top-level forests for each project, kinda like:
```
- Inbox
  - unfiled tasks
- Finances
  - subtasks
- Learn
  - Haskell
    - subtasks
  - GTD
    - subtasks
- Work
  - Company 1
    - subtasks
  - Company 2
    - etc
```

Is the recommended workflow to have `workflows/projects/{finances.smos,learn-haskell.smos,learn-gtd.smos,etc.}`?
Should we have separate "inboxes" for each project or is there a way to move entries between files/projects? 
Say I could have a `workflows/inbox.smos` and then when I'm in the sorting cycle, move them to the appropriate projects or to `workflows/general.smos` for non-project tasks?

Reply:
@prikhi I'm still working on the docs (blogposts) but feel free to write me emails with questions like these in the meantime. You do indeed want to have one project per file under `workflow/projects`.
Smos is not for inboxing though. For inboxing I recomment intray.eu